parm/config/config.base.emc.dyn

```diff
@@ -207,7 +207,8 @@ export OPS_RES="C768" # Do not change
 # Resolution specific parameters
 export LEVS=128
 export CASE="@CASECTL@"
-export CASE_ENKF="@CASEENS@"
+export CASE_ENKF="@CASEENS_ENKF@"
+export CASE_GEFS="@CASEENS_GEFS@"
```

Comment:
I am still not convinced we will have a use case for running GEFS that is a resolution different than the cycled ensemble resolution.

Reply:
@aerorahul Any suggestions? Should I change it back to 'export CASE_ENKF="@CASEENS@"' and the corresponding changes for the options or keep this changes?